src/main/java/com/kodilla/ecommercee/controller/UserController.java

```diff
@@ -1,24 +1,21 @@
 package com.kodilla.ecommercee.controller;
-
-
 import com.kodilla.ecommercee.domain.User;
 import com.kodilla.ecommercee.domain.UserDto;
 import com.kodilla.ecommercee.exception.UserNotFoundException;
 import com.kodilla.ecommercee.mapper.UserMapper;
-import com.kodilla.ecommercee.service.DbService;
+import com.kodilla.ecommercee.service.UserDbService;
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.MediaType;
 import org.springframework.web.bind.annotation.*;
 
-import java.time.LocalDateTime;
 import java.util.List;
 
 @RestController
 @RequiredArgsConstructor
 @RequestMapping("/v1/user")
 public class UserController {
     private final UserMapper userMapper;
-    private final DbService service;
+    private final UserDbService service;
 
     @RequestMapping(method = RequestMethod.GET, value = "getUsers")
     public List<UserDto> getUsers() {
```

Comment:
Tu gdybyśmy chcieli zostawić ten endpoint to lepiej by było chyba stworzyć DTO, które zawierałoby tylko id, username i status, bo userKey i expirationDate to są rzeczy, których raczej nie chcemy przekazywać "na zewnątrz"

Reply:
ExpirationDate nie jest w Dto a zgodnie ze specyfikacja endpoint zwraca ID, Username, status i userKey